docs/blocks/custom-fluids.md

```diff
@@ -0,0 +1,851 @@
+---
+title: Custom Fluids
+category: Tutorials
+tags:
+    - experimental
+    - intermediate
+mention:
+    - Provedule
+---
+
+:::warning
+**Requires** a basic knowledge of blocks and commands, including permutations, events, properties, components, and the new execute command.
+:::
+
+Creating fluids that are identical to vanilla fluids is not currently possible, but it is possible to make something similar. This template/tutorial is designed to assist you in creating a custom semi-fluid.
+
+## How Semi-fluids Work
+
+Semi-fluids consist of five blocks: three blocks form the outer parts of the fluid, one block acts as the main source of the fluid, and the final block represents the falling part of the fluid. Additionally, execute commands are utilized to identify blocks and establish rules for the fluid flow. Throughout the remainder of this guide, semi-fluids will be referred to as fluids.
+
+<WikiImage
+	src="/assets/images/blocks/custom-fluids/fluidDisplay.png"
+	alt="alternative text"
+	pixelated="true"
+	width=540
+/>
+
+## Source Fluid Block
+
+Below is the `source fluid block` code. To replicate the json, copy and quick replace `template` with your fluid's name. When the source block detects air in its surroundings, it replaces it with the `outer fluid block 1`. If the source block detects air beneath it, it will also place a `falling fluid block` underneath.
+
+<CodeHeader>BH\blocks\fluid_template\fluid_template.json</CodeHeader>
```

Comment:
Please make the codeheader follow the same style guide, `BP/blocks/filename.json`